docker-compose.yml

```diff
@@ -1,40 +1,40 @@
-version: "3"
```

Comment:
Can you reclone repo and reapply your changes. The files are being uploaded as CLRF.

Reply:
should be fixed